infrastructure/sdk.Dockerfile

```diff
@@ -51,6 +51,6 @@ RUN --mount=type=cache,id=composer,sharing=locked,target=/home/spryker/.composer
 
 ENV APP_ENV=prod
 
-RUN bin/console cache:warmup
+RUN bin/console cache:clear --no-debug
```

Comment:
I guess `cache:warmup` includes `cache:clear` inside f. or it's fix for issues with cache to use debug mode?

Reply:
`warmup` is included in cache clear

Reply:
The cache:clear command clears and warms up the application cache for a given environment
  and debug mode: